.github/workflows/build.yml

```diff
@@ -22,15 +22,15 @@ jobs:
 
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - name: Checkout ${{ github.event.pull_request.head.ref }}
-      uses: actions/checkout@v3
+      uses: actions/cache@v3.0.11
```

Comment:
Looking at https://github.com/actions/checkout/blob/main/CHANGELOG.md

This should probably be: `actions/checkout@v4`

Reply:
cache can also be updated to v4 `https://github.com/actions/cache` :) But you need to do that at the relevant line.